apps/web/src/pages/templates/components/StepEditorSidebar.tsx

```diff
@@ -33,6 +34,7 @@ export const StepEditorSidebar = ({ children }: { children: ReactNode }) => {
   }>();
   const navigate = useNavigate();
   const basePath = useBasePath();
+  const { navigateToVariantPreview } = useNavigateToVariantPreview();
```

Comment:
the hook that allows navigating to the variants preview page, is reused in many places...